samples/sql-persistence/saga-rename/SqlPersistence_7/EndpointVersion1/MyReplySagaVersion1.cs

```diff
@@ -16,8 +16,10 @@ public class MyReplySagaVersion1 :
 
         protected override void ConfigureHowToFindSaga(SagaPropertyMapper<SagaData> mapper)
         {
-            mapper.ConfigureMapping<StartReplySaga>(msg => msg.TheId).ToSaga(saga => saga.TheId);
-            mapper.ConfigureMapping<Reply>(msg => msg.TheId).ToSaga(saga => saga.TheId);
+            mapper.MapSaga(saga => saga.TheId)
+                .ToMessage<StartReplySaga>(msg => msg.TheId)
+                .ToMessage<Reply>(msg => msg.TheId)
+                ;
```

Comment:
Not a big fan of a sole-semi-colon on its own, but I acknowledge that's being subjective (I'm surprised the FXCop didn't catch you doing that!)

Reply:
Hah. It's habbit. Every time I need to add another mapping to the saga, I need to move the semi-colon so I tend to leave it on it's own for the next one. I don't feel strongly about it.